### PR TITLE
Add tracking stations used in launches from Kourou

### PR DESCRIPTION
**Pull request by leudaimon**
I probably did not use the correct naming convention (nor range), but the location is correct.

### CODE REVIEW

**Comment by PhineasFreak** [comment]
Yea, the station IDs should probably be ESTRACK (even if they are not listed by ESA as such but they support all their launches from Kourou). And they should be yellow (1,1,0,1) since they are tracking stations. The ranges are fine.

**Comment by NathanKell** [comment]
PhineasFreak you clearly know this stuff better than I, so go ahead and merge when you feel ready :)

**Comment by PhineasFreak** [comment]
@NathanKell I do not have write access to this repository.

File: GameData/RealSolarSystem/RemoteTech_Settings.cfg [hunk 1]
```diff
@@ -931,7 +931,60 @@
...
+	       ANTENNA
+	       {
+	           Omni = 5E+07
+		   }
+		}
+	}
+	STATION
+	{
+		Guid	  = 5105f5a9-d628-41c6-ad4b-21154e8fc999
+		Name	  = ESTRACK Ascencion
```
Comment (eggrobin) [inline GameData/RealSolarSystem/RemoteTech_Settings.cfg]: Typo (should be Ascension).

File: GameData/RealSolarSystem/RemoteTech_Settings.cfg [hunk 2]
```diff
@@ -954,7 +1007,7 @@
         // MSFN
         STATION
         {
-            Guid	  = 5106f5a9-d628-41c6-ad4b-21154e8fc100
+            Guid	  = 5106f5a9-d629-41c6-ad4b-21154e8fc100
```
Comment (eggrobin) [inline GameData/RealSolarSystem/RemoteTech_Settings.cfg:1028]: What's going on here?

**Review by eggrobin — COMMENTED** [review]
Do we want to add Galliot (about 20 km southwest of the ELAs, 30 km southwest of the ELS, across the river from Kourou)?
https://goo.gl/maps/AeJAmrBe6yM2

**Comment by eggrobin** [comment]
We definitely want Malindi, https://goo.gl/maps/zCpVyYYuLGt.
Together with Galliot, that would give us the whole network, [Galliot→Natal→Ascension→Libreville→Malindi](http://www.cnes-csg.fr/web/CNES-CSG-fr/9793-les-moyens-de-telemesure-et-localisation.php).

**Comment by leudaimon** [comment]
@eggrobin I'm on mobile, so I'll take a look at fixing the ascension thing later. 
About the guid , iirc I noticed there were two stations with identical IDs, so I just changed one. 
About Galliot, isn't it too close to kourou to matter?
Malindi is already in the file, just below the place where I added the other ones

**Comment by eggrobin** [comment]
Ah yes, I see Malindi. If we're counting Natal, Ascension, and Librevile as ESTRACK, should Malindi be like that too?
As for Galliot, the thing is the Kourou station in that file has a tiny range, whereas Galliot has the same sort of equipment as the others. If Kourou is in sight of Galliot in-game, perhaps the Kourou station could be removed and we could just keep Galliot, I'm pretty sure Galliot has telemetry from the start IRL: see [this document about VA226](http://www.airbus.com/content/dam/corporate-topics/publications/press-release/dossier-de-vol-v226-en.pdf), section 8, there is no acquisition Galliot.

**Comment by PhineasFreak** [comment]
@eggrobin It would probably be better to keep the Kourou station since every RSS launch site is also a tracking station.

**Comment by eggrobin** [comment]
A random thought from looking at this file, the comment at the beginning of the file is incorrect, since @NathanKell [changed the value from 250 km to 5000 km](https://github.com/leudaimon/RealSolarSystem/commit/fbc0ef295dceb998d6aa9f826df795f53739c5a9) without updating [the comment introduced](https://github.com/leudaimon/RealSolarSystem/commit/ddbaa89c065207e6c8f3892c6060218e83066439) by @jbengtson.

**Comment by eggrobin** [comment]
@PhineasFreak, that makes sense, but then we still need the longer-range station of Galliot (with a known dish, rather than just "is a launch site so 5000 km").

**Comment by PhineasFreak** [comment]
@eggrobin Oh, that's OK, more stations are always better! Probably misunderstood your comment...

**Comment by eggrobin** [comment]
Looks good to me

**Comment by NathanKell** [comment]
Merging. Also @PhineasFreak your perms are fixed. :)

**Pull request merged**